src/main/java/com/iexec/commons/containers/DockerLogs.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright 2023 IEXEC BLOCKCHAIN TECH
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.iexec.commons.containers;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+public class DockerLogs {
+
+    private String stdout;
+    private String stderr;
+
+}
```

Comment:
Not yours but still:
```
 * limitations under the License.**h** <-------- char to remove
 */
package com.iexec.commons.containers;

public enum DockerRunFinalStatus {
```

Reply:
Same in test file :S

Reply:
Please see https://github.com/iExecBlockchainComputing/iexec-commons-containers/pull/3/commits/b368497cbce907f8202a80c15267394bcd0a2925.